### ISSUE TITLE
How long does it take to train on CIFAR10?

### ISSUE THREAD

**Issue by guoqincode** [yanivbenny/dual_diffusion#1]
Hello, this is really an interesting work. I tried to use a RTX 3090 to train on CIFAR10. After 15h of training, 292 epcoh were trained, and val loss reached 0.0465.
I wonder how far I need to train to stop training?

**Comment by yanivbenny** [comment]
The validation loss is a poor estimate for progression since it averages all timesteps.
This is why you may see the validation loss stop improving while also see the model generate better results.
The longer you train the better, but there is a stop at 1,000,000 iterations.
